build

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+
+# Run unit tests and calculate code coverage.
+rm -f .coverage &&
+for dir in plsync ; do
+  PYTHONPATH=$PYTHONPATH:$PWD/site-packages coverage run --append --source ./ \
+      --omit third_party/docstringchecker/*.py \
+      -m unittest discover -s ${dir} -p '*_test.py' || exit 1
+done &&
```

Comment:
I find the `&&` so far separated from the following commands to be a little awkward. Rather than putting `&&` after blocks in this file, could you instead `set -e` at the top of the file?